### PR TITLE
MySQL - minor grammar

### PR DESCRIPTION
**Pull request by johnpapa**
#sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/mysql.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-23/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180429349125-126%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/mysql.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Learn more about [Azure Database for MySQL](https://docs.microsoft.com/azure/MyS
 
 ## Client Package
 
-### Install npm module
+### Install the npm module
 
 Use npm to install the MySQL client module.
```
Comment (clarkio) [inline docs-ref-services/mysql.md]: This says to install the `mysql` module, but the over view links to a different module which has documentation to install with a different module name: https://github.com/sidorares/node-mysql2#installation. Which one should it be?

Comment (johnpapa) [inline docs-ref-services/mysql.md]: @simonaco  please fix this. see brian's note above

Comment (simonaco) [inline docs-ref-services/mysql.md]: @clarkio good catch, thanks. Fixed in https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/128.

**Pull request merged**